sources/declarative/decodable/DecoderSequence.swift

```diff
@@ -0,0 +1,31 @@
+open class CSVDecoderSequence: IteratorProtocol, Sequence {
+    private let source: ShadowDecoder.Source
+    private var currentIndex: Int = 0
```

Comment:
Should we expose `currentIndex`? (`private(set) public var`) Do you see any value on this?

Reply:
Probably not. If you wanted the row index, you could use `enumerated()`.